cpp/CMakeLists.txt

```diff
@@ -153,7 +153,8 @@ rapids_cpm_init()
 
 # Putting this before raft to override RAFT from pulling them in.
 include(cmake/thirdparty/get_libcudacxx.cmake)
-include(cmake/thirdparty/get_cuco.cmake)
+include(${rapids-cmake-dir}/cpm/cuco.cmake)
```

Comment:
I think you need to do one more merge of branch-22.08 to get my cuco changes in here.